packages/xtee-misp2-postgresql/debian/changelog

```diff
@@ -1,3 +1,7 @@
+xtee-misp2-postgresql (2.9.0) stable; urgency=medium
+  * Fix fresh install issues on Ubuntu 22.04
+  * Drop support for Ubuntu 18.04
+
```

Comment:
It would be good to add metadata about the maintainer and release date/time like the other entries (except the previous three), for example:

> -- NIIS [info@niis.org](mailto:info@niis.org) Wed, 06 Apr 2022 15:38:05 +0300

Reply:
Added, also updated the older entries since they were causing warnings.